blog.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Adopt A Dog</title>
+    <link rel="stylesheet" type="text/css" href="styles/stylesheet.css">
+  </head>
+  <body>
+    <header>
+      <div class="float-right">
+        <img src="images/cart-icon.svg" alt="white icon outline of a shopping cart" width="40">
+        <button>Checkout</button>
+      </div>
+      <img class="logo" src="images/adoptadog-logo.svg" alt="adopt a dog logo of a white dog silhouette sitting with the adopt a dog name beside it" width="300">
+      <h1><em>Seattle's premiere dog adoption agency</em></h1>
+    </header>
+    <nav class="top-nav">
+      <ul>
+        <li><a href="index.html">Home</a></li>
+        <li>/</li>
```

Comment:
There are better ways to handle these / marks. The advanced (best) way would be to use a css pseudo element. You could research that (you'll need to learn how to use the after selector and content) or you could just add a slash after the Home and Dogs anchor tags